src/YesNoBooleanParser.php

```diff
@@ -4,6 +4,11 @@
 
 class YesNoBooleanParser extends AbstractValueParser
 {
+    protected function describe()
+    {
+        return "either yes or no";
```

Comment:
`return "either yes, no, Y or N";` might be the complete description, but then again, I guess it doesn't really matter :)